holoviews/core/util.py

```diff
@@ -235,24 +245,25 @@ def argspec(callable_obj):
     if (isinstance(callable_obj, type)
         and issubclass(callable_obj, param.ParameterizedFunction)):
         # Parameterized function.__call__ considered function in py3 but not py2
-        spec = inspect.getargspec(callable_obj.__call__)
-        args=spec.args[1:]
+        spec = _getargspec(callable_obj.__call__)
+        args = spec.args[1:]
     elif inspect.isfunction(callable_obj):    # functions and staticmethods
-        return inspect.getargspec(callable_obj)
+        spec = _getargspec(callable_obj)
+        args = spec.args[1:]
```

Comment:
@philippjfr I am not sure whether this should `spec.args[1:]` or just `spec.args`.

Reply:
It has been a while that I looked at this but I imagine that the main reason to slice like this would be to get rid of ``self`` on methods.